### ISSUE TITLE
Get body/freestyle libre temperature...

### ISSUE THREAD

**Issue by Kywalh** [NightscoutFoundation/xDrip#494]
Hello,
Just wondering if we can get the sensor/body temperature and display it on main screen....

**Comment by Kywalh** [comment]
No idea ?

**Comment by Navid200** [comment]
I don't think Libre has a temperature sensor, does it?

**Comment by tolot27** [comment]
The Libre sensors have a temperature sensor but the values are not provided over BLE or NFC.

**Comment by Navid200** [comment]
Thanks

I had been told that the Libre CGM was sensitive to temperature because it had no temperature compensation.
If there is a sensor, I don't understand why there is no temperature compensation.

**Comment by tolot27** [comment]
Indeed, there is a temperature compensation implemented in the firmware. But compensation likely fails for situations like warm or ice bath. xDrip cannot compensate raw values by temperature because the values are not exposed.

**Comment by tolot27** [comment]
It looks like it is possible.

**Comment by Navid200** [comment]
I'm going to convert this to a discussion.
The conversation can continue there.
